src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -64,34 +70,44 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 2345")]
         [InlineData("HTTP/A.1 200 OK")]
         [InlineData("HTTP/X.Y.Z 200 OK")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
-        //[InlineData("HTTP/0.1 200 OK")]
-        //[InlineData("HTTP/3.5 200 OK")]
```

Comment:
I think we should find a way to at least note the test cases in here that are being deleted.